client/src/pages/course/mentor/expel-student.tsx

```diff
@@ -9,16 +9,30 @@ import { CoursePageProps, StudentBasic } from 'services/models';
 
 function Page(props: CoursePageProps) {
   const courseId = props.course.id;
+  const roles = props.session.roles;
 
   const [form] = Form.useForm();
   const courseService = useMemo(() => new CourseService(courseId), [courseId]);
   const [loading, setLoading] = useState(false);
   const [students, setStudents] = useState([] as StudentBasic[]);
 
   useAsync(async () => {
-    const mentorStudents = await courseService.getMentorStudents();
-    const students = mentorStudents.filter(student => student.isActive);
-    setStudents(students);
+    if (roles[courseId] === 'student') {
```

Comment:
@forkollaider  please just add a button to profile page - no need to make mentor specific page available to students.